doc/nrf/security/tfm/tfm_supported_services.rst

```diff
@@ -165,16 +165,35 @@ The following table lists the isolation level support in Nordic Semiconductor's
 Limitations
 ***********
 
-The following list summarizes the limitations of TF-M in the |NCS|:
+The following sections summarize the limitations of TF-M in the |NCS|, organized by category.
+
+Core features
+=============
 
 * TF-M profiles are not supported.
-* From among the TF-M Services, Firmware Update service is not supported.
-* The following crypto modules or ciphers are not supported:
+* Isolation level 3 is not supported.
+* Second-stage bootloader is not supported.
```

Comment:
Nit but you can maybe you can expanded a bit on this. 
The second-stage bootloader from TF-M (BL2)  is not supported but Nordic uses its own version of Mcuboot (sdk-mcuboot) which is supported with TF-M.